analysis_driver/clarity/__init__.py

```diff
@@ -208,6 +208,15 @@ def get_run(run_id):
     if runs:
         return runs[0]
 
+def route_samples_to_delivery_workflow(sample_names):
+    lims = _get_lims_connection()
+    workflow_uri = lims.get_uri('configuration', 'workflows', '401')
+    samples = [get_lims_sample(sample_name, lims) for sample_name in sample_names]
+    artifacts = [sample.artifact for sample in samples]
+    lims.route_artifacts(artifacts, workflow_uri=workflow_uri)
+
```

Comment:
We could do named searching by iterating through the `workflows` endpoint (but would be mainly a fix in `genologics`)